test/models/user_test.rb

```diff
@@ -3,7 +3,13 @@
 require 'test_helper'
 
 class UserTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test '名前がないユーザはメールアドレスを返す' do
+    yocchan = build(:yocchan)
```

Comment:
前のコメントにも書きましたが、名前のあるなしはファクトリの定義を見に行かないとどういうデータが登録されているのかがわかりません。
それよりも、ファクトリボットの柔軟性を活用するようにした方がよいです。

```suggestion
    yocchan = build(:user, email: 'yocchan@example.com', name: '')
```